src/OrchardCore.Modules/OrchardCore.AdminTrees/Views/Node/List.cshtml

```diff
@@ -0,0 +1,135 @@
+                {
+                    if (menuItem != null)
+                    {
+                        dynamic adminNodeShape = await MenuItemDisplayManager.BuildDisplayAsync(menuItem, updater, "TreeSummary");
+                        adminNodeShape.AdminTreeId = Model.AdminTree.Id;
+                        adminNodeShape.MenuItem = menuItem;
+                        adminNodeShape.Index = index++;
+
+                        @await DisplayAsync(adminNodeShape)
+                    }
+                }
+            </ol>
+        }
+        else
+        {
+            <div class="alert alert-info" role="alert">
+                @T["<strong>Nothing here!</strong> There are no nodes on this tree for the moment."]
+            </div>
+        }
+    </div>
+
+    <div class="form-group">
+        <a class="btn btn-secondary" asp-route-controller="tree" asp-route-action="List">@T["Back"]</a>
+    </div>
+
+</form>
+
+<!-- Modal -->
+<div class="modal fade" id="modalTreeNodes" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">@T["Available Tree Nodes"]</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <div class="card-columns">
+                    @foreach (var thumbnail in Model.Thumbnails)
+                    {
+                        thumbnail.Value.ContentTreePreset = Model.AdminTree;
+                        thumbnail.Value.Type = thumbnail.Key;
+                        @await DisplayAsync(thumbnail.Value)
+                    }
+                </div>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">@T["Cancel"]</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+
+<script at="Foot">
+    function moveNode(AdminTreeId, nodeToMoveId, destinationNodeId, position) {
+        var url = "@Url.Action("MoveNode", "Node", new { area = "OrchardCore.AdminTrees"})";
+        $.ajax({
+            url: url,
+            method: 'POST',
+            data: {
+                __RequestVerificationToken: $("input[name='__RequestVerificationToken']").val(),
+                treeId: AdminTreeId,
+                nodeToMoveId: nodeToMoveId,
+                destinationNodeId: destinationNodeId,
+                position: position
+            },
+            success: function () {
+                location.reload(true);
+            },
+            error: function (error) {
+                console.error(error.responseText);
+
+                alert("@T["There was an error when moving the tree node"]");
```

Comment:
Don't use localization in JS, create a hidden div with a data attribute containing the text, and reference it using js. Otherwise a translation might have a `"` and break the JavaScript.